agent/src/agent/source/validator.py

```diff
@@ -147,6 +150,24 @@ def validate_connection_string(self):
             raise ValidationException('Wrong url scheme. Use `clickhouse`')
 
 
+class OracleValidator(Validator):
+    VALIDATION_SCHEMA_FILE = 'jdbc.json'
+
+    def validate(self):
+        self.validate_json()
+        self.validate_connection_string()
+        self.validate_connection()
```

Comment:
you can inherit JDBCValidator and remove VALIDATION_SCHEMA_FILE and def validate(self):